.gitignore

```diff
@@ -86,6 +86,7 @@
 **/vendor/bundle/ruby/*/gems/jaro_winkler-*/
 **/vendor/bundle/ruby/*/gems/json-*/
 **/vendor/bundle/ruby/*/gems/json_schemer-*/
+**/vendor/bundle/ruby/*/gems/kramdown-*/
```

Comment:
Can we remove the entries for ronn and its dependencies?

Reply:
Yeah probably, will clean it up.